docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
+Default: `true`
+
+Allow logging in by email and password, when SSO login options are enabled.
+
+#### `MB_ENABLE_PUBLIC_SHARING`
+
+Type: boolean<br>
+Default: `false`
+
+Enable admins to create publicly viewable links (and embedded iframes) for Questions and Dashboards?
+
+#### `MB_ENABLE_QUERY_CACHING`
+
+Type: boolean<br>
+Default: `false`
+
+Enabling caching will save the results of queries that take a long time to run.
+
+#### `MB_ENABLE_TEST_ENDPOINTS`
+
+Type: boolean<br>
+Default: `null`
+
+When `true`, this will enable `/api/testing` endpoint. **Warning:** This should never be enabled in production system.
+
+#### `MB_ENABLE_XRAYS`
+
+Type: boolean<br>
+Default: `true`
+
+Allow users to explore data using X-rays
+
+#### `MB_ENCRYPTION_SECRET_KEY`
+
+Type: string<br>
+Default: `null`
+
+When set, this will encrypt database credentials stored in the application database. Requirement: minimum 16 characters base64-encoded string.
+
+Also see documentation page [Encrypting database details at rest](encrypting-database-details-at-rest.md).
+
+#### `MB_GOOGLE_AUTH_AUTO_CREATE_ACCOUNTS_DOMAIN`
+
+Type: string<br>
+Default: `null`
+
+When set, allow users to sign up on their own if their Google account email address is from this domain.
+
+#### `MB_GOOGLE_AUTH_CLIENT_ID`
+
+Type: string<br>
+Default: `null`
+
+Client ID for Google Auth SSO. If this is set, Google Auth is considered to be enabled.
+
+#### `MB_JDBC_DATA_WAREHOUSE_MAX_CONNECTION_POOL_SIZE`
+
+Type: integer<br>
+Default: `15`<br>
+Since: 0.35.0
+
+Maximum number of connections to the data source databases. The maximum is for each database setup in Admin Panel > Databases, not a total for all databases.
+
+Change this to a higher value, if you notice that regular usage consumes close to or all connections. When all connections are in use, then Metabase will be slower to return results for queries, since it would have to wait for an available connection before processing the next query in the queue.
+
+See [MB_APPLICATION_DB_MAX_CONNECTION_POOL_SIZE](#MB_APPLICATION_DB_MAX_CONNECTION_POOL_SIZE) for setting maximum connections to the Metabase application database.
+
+#### `MB_JETTY_ASYNC_RESPONSE_TIMEOUT`
+
+Type: integer<br>
+Default: `600000`<br>
+Since: 0.35.0
+
+Timeout of Jetty async threads, defined in milliseconds. The default is 10 minutes. Very few things might reach the timeout, since they return some type of data before, but things like CSV might.
+
+#### `MB_JETTY_DAEMON`
+
+Type: boolean<br>
+Default: `false`
+
+Use daemon threads.
+
+#### `MB_JETTY_HOST`
+
+Type: string<br>
+Default: `null`
+
+Configure a host either as a host name or IP address to identify a specific network interface on which to listen. If not set or set to `"0.0.0.0"`, Metabase listens on all network interfaces. It will listen on the port specified in [MB_JETTY_PORT](#MB_JETTY_PORT).
+
+#### `MB_JETTY_JOIN`
+
+Type: boolean<br>
+Default: `true`
+
+Blocks the thread until server ends.
```

Comment:
Not sure I understand this. "Server ends?"

Reply:
I have no idea what this does - spend way too long trying to figure it out - it's a comment copied from Ring.
Almost positively sure that you shouldn't touch this unless testing something.